manifests/init.pp

```diff
@@ -115,6 +120,7 @@
   Stdlib::Fqdn $servername = $facts['networking']['fqdn'],
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   Optional[String] $remote_user_environ_name = undef,
+  Integer[1] $worker_count = min(8, $facts['processors']['count']),
```

Comment:
Just thinking out loud: for the future it may actually make sense that you want to deploy a server that's only a webserver and do the worker processing elsewhere. So is `0` a better lower bound?